SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md

```diff
@@ -2,9 +2,8 @@ The CBL-Mariner SPEC files originated from a variety of sources with varying lic
 pkg-config <br> polkit <br> popt <br> postgresql <br> powershell <br> procps-ng <br> protobuf <br> protobuf-c <br> psmisc <br> pth <br> pyOpenSSL <br> pyasn1-modules <br> pygobject3 <br> pyparsing <br> pytest <br> python-PyNaCl <br> python-Pygments <br> python-Twisted <br> python-appdirs <br> python-asn1crypto <br> python-atomicwrites <br> python-attrs <br> python-backports-ssl_match_hostname <br> python-bcrypt <br> python-boto3 <br> python-botocore <br> python-certifi <br> python-cffi <br> python-chardet <br> python-configobj <br> python-constantly <br> python-coverage <br> python-cryptography <br> python-daemon <br> python-dateutil <br> python-defusedxml <br> python-distro <br> python-docopt <br> python-docutils <br> python-ecdsa <br> python-enum34 <br> python-futures <br> python-gevent <br> python-greenlet <br> python-hyperlink <br> python-hypothesis <br> python-idna <br> python-imagesize <br> python-incremental <br> python-iniparse <br> python-ipaddr <br> python-ipaddress <br> python-jinja2 <br> python-jmespath <br> python-jsonpatch <br> python-jsonpointer <br> python-jsonschema <br> python-lockfile <br> python-lxml <br> python-m2r <br> python-mako <br> python-markupsafe <br> python-mistune <br> python-msgpack <br> python-netaddr <br> python-netifaces <br> python-ntplib <br> python-oauthlib <br> python-packaging <br> python-pam <br> python-pbr <br> python-pip <br> python-ply <br> python-prettytable <br> python-psutil <br> python-psycopg2 <br> python-py <br> python-pyasn1 <br> python-pycodestyle <br> python-pycparser <br> python-pycurl <br> python-pyvmomi <br> python-requests <br> python-setuptools <br> python-setuptools_scm <br> python-simplejson <br> python-six <br> python-snowballstemmer <br> python-sphinx <br> python-sphinx-theme-alabaster <br> python-sqlalchemy <br> python-typing <br> python-urllib3 <br> python-vcversioner <br> python-virtualenv <br> python-wcwidth <br> python-webob <br> python-websocket-client <br> python-werkzeug <br> python-zope-interface <br> python2 <br> python3 <br> pytz <br> rapidjson <br> readline <br> redis <br> rng-tools <br> rpcbind <br> rpcsvc-proto <br> rpm <br> rpm-ostree <br> rrdtool <br> rsync <br> rsyslog <br> ruby <br> rubygem-bundler <br> runc <br> rust <br> scons <br> sed <br> sg3_utils <br> shadow-utils <br> slang <br> snappy <br> socat <br> sqlite <br> sshpass <br> strace <br> strongswan <br> subversion <br> sudo <br> swig <br> syslinux <br> syslog-ng <br> sysstat <br> systemd <br> systemtap <br> tar <br> tboot <br> tcl <br> tcp_wrappers <br> tcpdump <br> tcsh <br> tdnf <br> telegraf <br> texinfo <br> tmux <br> tpm2-abrmd <br> tpm2-tools <br> tpm2-tss <br> traceroute <br> tree <br> trousers <br> tzdata <br> unbound <br> unixODBC <br> unzip <br> usbutils <br> userspace-rcu <br> utf8proc <br> util-linux <br> valgrind <br> vim <br> vsftpd <br> wget <br> which <br> wpa_supplicant <br> xerces-c <br> xfsprogs <br> xinetd <br> xmlsec1 <br> xmlto <br> xz <br> zchunk <br> zeromq <br> zip <br> zlib <br> zsh |
```

Comment:
I see runc in this list, runc is removed from dev.

Reply:
thanks, removed. this file was a difficult merge